tirgei/Questioner
# #163788846 Consume endpoint to fetch upcoming meetups

#### What does this PR do?
This Pull Request consumes the endpoint to fetch all upcoming meetups in the next 1 week

#### Description of Task to be completed?
The tasks for this PR is:
- Add function to fetch upcoming meetups
- Add empty state for when no meetups is happening in the next week
- Add list to display upcoming meetups

#### How should this be manually tested?
Clone this repository on your machine:
```$ git clone https://github.com/tirgei/Questioner.git ```

CD into the project folder on your machine
```$ cd Questioner ```

Check out the branch for this PR
``` $ git checkout ft-upcoming-meetups-163788846```

Navigate to the`index.html` file

#### Pivotal Tracker story associated with this PR
[#163788846](https://www.pivotaltracker.com/story/show/163788846)

#### Screenshots

No upcoming meetups
![upcoming-empty-state](https://user-images.githubusercontent.com/17829576/52483558-4eaca600-2bc5-11e9-88af-90a943f5e6de.png)

Upcoming meetup
![upcoming-meetups](https://user-images.githubusercontent.com/17829576/52483573-54a28700-2bc5-11e9-8213-aa9eed20e3e2.png)
